### DISCUSSION TITLE
Embeddded vision with icicle kit

### DISCUSSION THREAD

**Discussion by chepredir** [polarfire-soc/polarfire-soc-documentation#36] [How to] [answered]
Hello, 
I am trying a embedded vision project on the icicle kit.
I want to deploy a software in icicle kit that I have written in C, the software uses basic file handling  to read and write from it. I want the program to execute on the board. I have flashed latest reference design and yocto linux image.
Thank you

**Comment by hughbreslin** [comment]
Hi @chepredir would you be able to give a bit more info on what you're trying to achieve / where you're having an issue? Is the question regarding building your software on the board? 

Thanks,
Hugh

**Comment by sanjeevb-bansal** [comment]
FYI, Microchip is soon going to release new PFSoC device based Smart Embedded Vision Platform which is better compare to Icicle kit for embedded vision project. It will have readymade examples for imaging and video  capture, processing and display etc. : -

Pls bookmark below link for more updates in coming future:- 
https://www.microchip.com/en-us/solutions/industrial/smart-embedded-vision

**Comment by hughbreslin** [comment, marked as answer]
Marking this discussion as answered as there hasn't been any follow up. Please un-mark this selected answer and post a follow up query if required.
